Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py

```diff
@@ -177,12 +178,13 @@ def fetch_incidents(aws_client, aws_queue_url, max_fetch, parse_body_as_json):
             for message in messages["Messages"]:
                 receipt_handles.append(message['ReceiptHandle'])
                 incidents.append(parse_incident_from_finding(message, parse_body_as_json))
-                if len(incidents) == max_fetch:
+                incidents_created += 1
+                if incidents_created == max_fetch:
                     break
 
-        demisto.incidents(incidents)
-        for receipt_handle in receipt_handles:
-            client.delete_message(QueueUrl=aws_queue_url, ReceiptHandle=receipt_handle)
+            demisto.incidents(incidents)
+            for receipt_handle in receipt_handles:
```

Comment:
@jrauen 
it's available to delete messages during the `VisibilityTimeout` ?

Reply:
Yes. From the documentation:
```
Immediately after a message is received, it remains in the queue. To prevent other consumers from processing the message again, Amazon SQS sets a visibility timeout, a period of time during which Amazon SQS prevents other consumers from receiving and processing the message. The default visibility timeout for a message is 30 seconds.
```

Source: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html